openvpn/session/validator_mocks.go

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package session
+
+import (
+	"github.com/mysterium/node/identity"
+	"github.com/mysterium/node/session"
+)
+
+const mockedVPNConfig = "config_string"
+
+type mockedConfigProvider func() string
+
+func (mcp mockedConfigProvider) ProvideServiceConfig() (session.ServiceConfiguration, error) {
+	return mcp(), nil
+}
+
+func provideMockedVPNConfig() string {
+	return mockedVPNConfig
+}
```

Comment:
Why do we need this function, can't we take `const` directly?

Reply:
We cannot use plain string here since 'session.NewManager()' requires function as an argument where it is used in TestValidateReturnsFalseWhenNoSessionFound()